packages/connectivity/src/scp-cf/destination/destination-service.spec.ts

```diff
@@ -175,6 +211,36 @@ describe('destination service', () => {
       });
     });
 
+    it('returns if some destinations are broken - subaccount destinations', async () => {
```

Comment:
```suggestion
    it('only returns valid destinations - subaccount destinations', async () => {
```
[req] also for the other test